tests/build/build_templates.go

```diff
@@ -203,27 +202,49 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 					pipelineRun, err = kubeadminClient.HasController.GetComponentPipelineRun(componentNames[i], applicationName, testNamespace, "")
 					Expect(err).ShouldNot(HaveOccurred())
 				})
+
 				It("should have Pipeline Records", func() {
-					// Verify Records
 					records, err := resultClient.GetRecords(testNamespace, string(pipelineRun.GetUID()))
+					// temporary logs due to RHTAPBUGS-213
+					GinkgoWriter.Printf("records for PipelineRun %s:\n%s\n", pipelineRun.Name, records)
+					GinkgoWriter.Printf("got error getting records: %v\n", err)
 					Expect(err).NotTo(HaveOccurred())
 					Expect(len(records.Record)).NotTo(BeZero(), "No records found for PipelineRun %s", pipelineRun.Name)
 				})
 
 				It("should have Pipeline Logs", func() {
 					// Verify if result is stored in Database
+					// temporary logs due to RHTAPBUGS-213
 					logs, err := resultClient.GetLogs(testNamespace, string(pipelineRun.GetUID()))
+					GinkgoWriter.Printf("logs for PipelineRun %s:\n%s\n", pipelineRun.Name, logs)
+					GinkgoWriter.Printf("got error getting logs: %v\n", err)
```

Comment:
Would not it be better/cleaner to add annotation to error output to print it out only if error actually occurs?

https://onsi.github.io/gomega/#annotating-assertions